compose/lite/configuration.yml

```diff
@@ -0,0 +1,91 @@
+###############################################################
+#                   Authelia configuration                    #
+###############################################################
+
+host: 0.0.0.0
+port: 9091
+
+logs_level: debug
+
+# This secret can also be set using the env variables AUTHELIA_JWT_SECRET
+jwt_secret: a_very_important_secret
+
+default_redirection_url: https://auth.example.com
```

Comment:
I would name it public.example.com instead since this auth is bypassed for this domain. It would make it clearer.

Reply:
We may want to add another container for public.example.com so there is all 3 examples using the traefik middleware (authelia). I've not used it but this may be a good container to use: https://hub.docker.com/r/yeasy/simple-web/